linode/linode_client.py

```diff
@@ -283,6 +283,22 @@ def assign_ips(self, region, *assignments):
             ips.append(i)
 
         return ips
+    
+    def add_ip(self, linode_id):
+        if not linode_id:
```

Comment:
This library generally accepts inputs as objects or IDs, and if given an object take the ID out before passing it in, as seen [here](https://github.com/linode/python-linode-api/blob/master/linode/linode_client.py#L421).  Otherwise, I lean on the api to provide error messages - if they pass `None` in for `linode_id`, the API will let you know this is needed.  This keeps error messages consistent and allows users to test for APIErrors everywhere.